basiclti/basiclti-common/src/java/org/sakaiproject/util/foorm/Foorm.java

```diff
@@ -1534,17 +1537,30 @@ public LTISearchData secureSearch(String search, String vendor) {
 					searchValue = searchValue.replace(LTIService.LTI_SEARCH_TOKEN_DATE, "");
 					if(StringUtils.isNotEmpty(searchValue)) {
 						try {
+							String operator = "=";
+							// Support more searching on dates.
+							if (searchValue.startsWith("<")) {
+								operator = "<";
+								searchValue = searchValue.substring(1);
+							} else if (searchValue.startsWith(">")) {
+								operator = ">";
+								searchValue = searchValue.substring(1);
+							}
 							DateFormat df = DateFormat.getDateTimeInstance(DateFormat.MEDIUM, DateFormat.MEDIUM, rb.getLocale());
 							Date d = df.parse(searchValue);
-							
+
 							DateFormat sql_df = new SimpleDateFormat(LTIService.LTI_SEARCH_INTERNAL_DATE_FORMAT);
 							if ( "oracle".equals(vendor) ) {
-								sb.append(searchField + " = TO_DATE('"+sql_df.format(d)+"', 'DD/MM/YYYY HH24:MI:SS')");
+								sb.append(searchField + " "+operator+ " TO_DATE('"+sql_df.format(d)+"', 'DD/MM/YYYY HH24:MI:SS')");
```

Comment:
That's what I'm trying todo as I have a date and so I pass the format and the date string to the DB and get it to convert it to a date. This way the locale of the DB shouldn't be involved.